project/DistributionPackage.scala

```diff
@@ -232,110 +239,119 @@ object DistributionPackage {
     log: Logger,
     env: Map[String, String] = Map.empty
   ): Unit = {
-    for {
-      libMajor <- stdLibRoot.listFiles()
-      libName  <- (stdLibRoot / libMajor.getName).listFiles()
-    } yield {
-      indexStdLib(
-        libName,
-        stdLibVersion,
-        ensoVersion,
-        javaOpts,
-        cacheFactory,
-        log,
-        env
+    val modifiedLibs: ArrayBuffer[File] = ArrayBuffer()
+    for (libName <- stdLibRoot.listFiles()) {
```

Comment:
if the root remains to be `distribution/lib`, then this code would become:
```suggestion
    for (libNamespace <- stdLibRoot.listFiles()) {
       for (libName <- new File(stdLibRoot, libNamespace).listFiles()) {
```
e.g. read namespaces first, and then read the library name.s

Reply:
Done in a9a42b138d3b50fa0f240e0ba7c32db4981dd58b